cmd/symbols/build.sh

```diff
@@ -164,6 +166,18 @@ function buildSymbolsDockerImage() {
     echo "Building the $IMAGE Docker image... done"
 }
 
+# Builds the ctags docker image, used by universal-ctags-dev and the symbols Docker image.
+function buildCtagsDockerImage() {
+    ctagsDockerBuildContext="$(mktemp -d)"
+    trap "rm -rf $ctagsDockerBuildContext" EXIT
```

Comment:
```suggestion
    trap "rm -rf -- $ctagsDockerBuildContext" EXIT
```

(averts the risk of bad glob expansion adding additional flags)

Reply:
I plan to break up this script a bit more in a separate effort.